packages/usa-character-count/src/index.js

```diff
@@ -28,13 +35,25 @@ const getCharacterCountElements = (inputEl) => {
     throw new Error(`${INPUT} is missing outer ${CHARACTER_COUNT}`);
   }
 
+  const formGroupEl = characterCountEl.querySelector(FORM_GROUP);
+
+  if (!formGroupEl) {
+    throw new Error(`${CHARACTER_COUNT} is missing inner ${FORM_GROUP}`);
+  }
+
+  const labelEl = characterCountEl.querySelector(LABEL);
+
+  if (!labelEl) {
+    throw new Error(`${CHARACTER_COUNT} is missing inner ${LABEL}`);
+  }
```

Comment:
**For form-group:**
This might be a good thing to discuss when normalizing things like form group and when we're using it. 

Since we include this error state as a behavior for character count, I see the importance of flagging that it should be included so users are properly updated that their form is in the error state. 

Maybe it would be better to issue a warning instead? I was matching the existing behaviors by adding the error.

**For label:**
If a user doesn't include a label for character count, it would be considered an accessibility failure so I think it's important to include _but_ looking at it again, I think I should have used the `for` attribute and the character count's ID instead of checking if it exists within the element tag.

That way if a user included a label but didn't nest it within the `usa-character-count` div, it'll still pass the test. 

How do you feel about that?

Reply:
Went ahead and removed the error messages and related unit tests! We can rely on guidance for the proper markup structure

Reply:
Unit tests removed in fe4211882a16f30e80c4559bb970e9b3119dfdf4.